src/main/java/com/google/sps/servlets/YoutubePost.java

```diff
@@ -34,6 +34,18 @@ public static String getUrl(String id) {
     return "https://www.youtube.com/watch?v=" + id;
```

Comment:
Please make this non-static as well, and ideally it'd be great if the getter just return `this.url`, but the formatting is done on line 30.